docs/source/get_started/new_project.md

```diff
@@ -36,18 +41,25 @@ So, if you enter "Get Started", the folder for the project (`repo_name`) is auto
 | The Python package name for the project (short, all-lowercase) | `python_package` | `get_started` |
 
 
-The output of `kedro new` is a directory containing all the project files and subdirectories required for a basic Kedro project, ready to extend with the code.
+After you enter `kedro new`, the command line interface will ask which tools you'd like to include, which will depend on the requirements of your project. The options are [described in more detail above](#tools-to-configure-the-new-project) and in the [documentation about the new project tools](../starters/new_project_tools.md).
+
+The output of `kedro new` is a directory containing all the project files and subdirectories required. The exact code added for the project will depend on which tools you selected.
 
-## Create a new project from a configuration file
+### Create a basic project from a configuration file
 
-To customise a new project's directory and package name, use a configuration file to specify those values. The configuration file must contain:
+To create a new project with custom directory and package names, navigate to your preferred directory and type the following command, which passes in a configuration file `config.yml`:
+
+```bash
+kedro new --config=<path>/config.yml
+```
+The configuration file must contain:
 
 -   `output_dir` The path in which to create the project directory
 -   `project_name`
 -   `repo_name`
 -   `python_package`
```

Comment:
Add-ons are also configurable here - will double check what happens when they are omitted

Reply:
Thanks. I'll need some insight on how to include these so will await the changes on your branch for guidance.